scipy/optimize/tests/test_linprog.py

```diff
@@ -750,7 +750,7 @@ def test_zero_column_2(self):
                       method=self.method, options=self.options)
         _assert_unbounded(res)
         # Unboundedness detected in presolve
-        if self.options.get('presolve', True):
+        if self.options.get('presolve', True) and "highs" not in self.method:
```

Comment:
This fails for HiGHS now? It is a very simple problem that should be handled in presolve.

Reply:
See upstream issue https://github.com/ERGO-Code/HiGHS/issues/648

Reply:
Marked as xfail for highs methods